crates/biome_cli/src/execute/std_in.rs

```diff
@@ -169,7 +169,20 @@ pub(crate) fn run<'a>(
                 only,
                 skip,
             })?;
-            diagnostics.extend(result.diagnostics);
+            let content = match biome_path.extension_as_str() {
+                Some("astro") => AstroFileHandler::input(&new_content),
+                Some("vue") => VueFileHandler::input(&new_content),
+                Some("svelte") => SvelteFileHandler::input(&new_content),
+                _ => &new_content,
```

Comment:
I'm just curious about the differences between Astro, Vue, Svelte, and other diagnostics?

Reply:
We partially support them; we only able to inspect the JS **portion** of those files, so extract it from the whole file:

```vue
<script>
	let a; // we extract this portion of the file
</script>
```